conda_build/build.py

```diff
@@ -685,13 +694,16 @@ def build(m, config, post=None, need_source_download=True, need_reparse_in_env=F
 
         get_build_metadata(m, config=config)
         create_post_scripts(m, config=config)
-        create_entry_points(m.get_value('build/entry_points'), config=config)
+
+        if str(m.get_value('build/noarch')).lower() != "python":
+            create_entry_points(m.get_value('build/entry_points'), config=config)
         files2 = prefix_files(prefix=config.build_prefix)
 
         post_process(sorted(files2 - files1),
-                        prefix=config.build_prefix,
-                        config=config,
-                        preserve_egg_dir=bool(m.get_value('build/preserve_egg_dir')))
+                     prefix=config.build_prefix,
+                     config=config,
+                     preserve_egg_dir=bool(m.get_value('build/preserve_egg_dir')),
+                     noarch= m.get_value('build/noarch'))
```

Comment:
extra space here between = and m.  Are you running flake8?  I think pycharm can do it on the fly for you.